packages/website/blog/2023-03-13-announcing-typescript-eslint-v6-beta.md

```diff
@@ -0,0 +1,309 @@
+  - `TSIndexSignature`, `TSMethodSignature`, and `TSPropertySignatureBase` no longer have an `export` property.
+  - `TSPropertySignatureBase` no longer has an `initializer` property.
+- [fix(typescript-estree): account for namespace nesting in AST conversion](https://github.com/typescript-eslint/typescript-eslint/pull/6272): Namespaces with qualified names like `Abc.Def` now use a `TSQualifiedName` node, instead of a nested body structure.
+
+### ⏳ Errors on Invalid AST Parsing
+
+:::note
+These changes only impact API consumers of typescript-eslint that work at parsing level.
+If the extent of your API usage is writing custom rules, these changes don't impact you.
+:::
+
+The `@typescript-eslint/typescript-estree` parser is by default very forgiving of invalid ASTs.
+If it encounters invalid syntax, it will still attempt create an AST if possible: even if required properties of nodes don't exist.
+
+For example, this snippet of TypeScript code creates a `ClassDeclaration` whose `id` is `null`:
+
+```ts
+export class {}
+```
+
+Invalid parsed ASTs can cause problems for downstream tools expecting AST nodes to adhere to the ESTree spec.
+ESLint rules in particular tend to crash when given invalid ASTs.
+
+`@typescript-eslint/typescript-estree` will now throw an error when it encounters a known invalid AST such as the `export class {}` example.
+This is generally the correct behavior for most parsing contexts so downstream tools don't have to work with a potentially invalid AST.
+
+For consumers that don't want the updated behavior of throwing on invalid ASTs, a new `allowInvalidAST` option exists to disable the throwing behavior.
+Keep in mind that with it enabled, ASTs produced by typescript-eslint might not match their TSESTree type definitions.
+
+For more information, see:
+
+- The backing issue: [Parsing: strictly enforce the produced AST matches the spec and enforce most "error recovery" parsing errors](https://github.com/typescript-eslint/typescript-eslint/issues/1852)
+- The implementing pull request: [feat(typescript-estree): added allowInvalidAST option to throw on invalid tokens](https://github.com/typescript-eslint/typescript-eslint/pull/6247)
+
+### Other Developer-Facing Breaking Changes
+
+- [feat: remove semantically invalid properties from TSEnumDeclaration, TSInterfaceDeclaration and TSModuleDeclaration](https://github.com/typescript-eslint/typescript-eslint/pull/4863): Removes some properties from those AST node types that should generally not have existed to begin with.
+- [fix(utils): removed TRuleListener generic from the createRule](https://github.com/typescript-eslint/typescript-eslint/pull/5036): Makes `createRule`-created rules more portable in the type system.
+- [feat(utils): remove (ts-)eslint-scope types](https://github.com/typescript-eslint/typescript-eslint/pull/5256): Removes no-longer-useful `TSESLintScope` types from the `@typescript-eslint/utils` package.
```

Comment:
Might be good to mention that the alternative is to use `@typescript-eslint/scope-manager`